### PR TITLE
chore: Add Matt Giuca as co-editor (closes #650)

### LINKED ISSUE #650: Add Matt Giuca as Co-Editor
@chaals, @LJWatson (Chairs of the Web Platform WG): As is required by the W3C process,  the Editors would like to request @mgiuca become a co-editor of the Web Manifest specification. 

Over the last year, @mgiuca has made significant contributions to the specification ([commit history](https://github.com/w3c/manifest/commits?author=mgiuca), and plans to continue to help us move this specification to W3C Recommendation status. 
 
On behalf of the editors, we would like to acknowledge his substantial contributions and continued participation in the specification by listing him as co-editor.

### PR DESCRIPTION
**Pull request by marcoscaceres**
* closes #650

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/pull/651.html" title="Last updated on Feb 6, 2018, 1:58 AM GMT (1a98bb9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/651/c586cf0...1a98bb9.html" title="Last updated on Feb 6, 2018, 1:58 AM GMT (1a98bb9)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,17 @@
...
+      }, {
+        name: "Kenneth Rohde Christiansen",
+        company: "Intel Corporation",
+        companyURL: "https://intel.com/",
+        w3cid: 57705
+      }, {
+        name: "Matt Giuca",
+        company: "Google Inc.",
+        companyURL: "https://google.com",
+        w3cid: 00000
```
Comment (marcoscaceres) [inline index.html]: Need proper ID here.

**Comment by marcoscaceres** [comment]
Need ok from the chairs before merging.

File: index.html [hunk 2]
```diff
@@ -40,7 +40,17 @@
         company: "Microsoft Corporation",
         companyURL: "https://microsoft.com/",
         w3cid: 60712
-      },],
+      }, {
+        name: "Kenneth Rohde Christiansen",
+        company: "Intel Corporation",
+        companyURL: "https://intel.com/",
+        w3cid: 57705
+      }, {
```
Comment (mounirlamouri) [inline index.html]: Did Kenneth duplicae? :)

Comment (marcoscaceres) [reply index.html]: LOL, Kenneth puts in so much work we need to list him twice!

**Comment by marcoscaceres** [comment]
Fixed the copypasta 🍝.

**Comment by mgiuca** [comment]
lgtm with changes requested.

Thanks for hooking me up, Marcos! Looking forward to official Manifest editing.

**Pull request merged**